lib/shopify_app/configuration.rb

```diff
@@ -50,13 +50,8 @@ def login_url
     end
 
     def session_repository=(klass)
-      if Rails.configuration.cache_classes
-        ShopifyApp::SessionRepository.storage = klass
-      else
-        ActiveSupport::Reloader.to_prepare do
-          ShopifyApp::SessionRepository.storage = klass
-        end
-      end
+      @session_repository = klass
```

Comment:
`attr_accessor` should be changed to `attr_reader` it is probably raising a warning as well already because you the method is being redefined.